src/plugins/LabelBot/strategies/typeOfChange.ts

```diff
@@ -5,30 +5,30 @@ import { extractTasks } from "../../../util/text_parser";
 const BODYMATCHES = [
   {
     description: "Bugfix (non-breaking change which fixes an issue)",
-    labels: ["type: bugfix"],
+    labels: ["bugfix"],
   },
   {
     description: "Dependency upgrade",
-    labels: ["type: dependency"],
+    labels: ["dependency"],
```

Comment:
We also have [`dependency-bump`](https://github.com/home-assistant/core/labels/dependency-bump).
Only one of them is probably needed, but that will be part of the cleanup after this PR.